### PR TITLE
Make accessible Windows build number

### PR DESCRIPTION
**Pull request by skvl**
For https://github.com/tklengyel/drakvuf/pull/766

### CODE REVIEW

File: libvmi/accessors.c [hunk 1]
```diff
@@ -95,6 +95,34 @@ vmi_get_winver(
 #endif
 }
 
+uint16_t
+vmi_get_win_buildnumber(
```
Comment (tklengyel) [inline libvmi/accessors.c]: The only issue I see is that this right now only works if a JSON profile was provided, in all other cases the value returned would just be 0. I would add a safety check to see if LibVMI was initialized with a Json profile or not (check if `vmi->json.handler == NULL` and print an error if it wasn't.

Comment (skvl) [reply libvmi/accessors.c]: @tklengyel, done!

File: libvmi/accessors.c [hunk 2]
```diff
@@ -95,6 +95,45 @@ vmi_get_winver(
...
+    errprint("**LibVMI wasn't compiled with Windows support!\n");
+    return 0;
+#else
+
+#ifndef ENABLE_JSON_PROFILES
+    errprint("**LibVMI wasn't compiled with JSON profiles support!\n");
+    return 0;
+#endif
+
+    if (vmi->json.handler == NULL) {
```
Comment (tklengyel) [inline libvmi/accessors.c]: This should be moved to be under ` #ifdef ENABLE_SAFETY_CHECKS `

**Comment by tklengyel** [comment]
@drakvuf-jenkins This is OK to test

File: libvmi/accessors.c (hunk 2 above)
Comment (skvl) [reply libvmi/accessors.c]: @tklengyel, done!

**Pull request merged**